crt_portal/static/js/complaint_view_filters.js

```diff
@@ -223,6 +224,25 @@
     return options.filter(isSelected).map(unwrapValue);
   };
 
+  function checkBoxView(props) {
+    props.el.forEach(function(element) {
+      element.addEventListener('change', function(event) {
+        checkBoxView.getValues(event.target);
+      });
+    });
+  }
+
+  checkBoxView.getValues = function(el) {
+    if (el.checked) {
+      filterDataModel[event.target.name].push(el.value);
+      return el.value;
+    } else {
+      var index = filterDataModel[event.target.name].indexOf(el.value);
+      filterDataModel[event.target.name].splice(index, 1);
+      return el.value;
+    }
+  };
```

Comment:
Did we test in IE11/Edge? If not, I can do this myself, just wanted to verify that this code would work. I don't think anything here will break, but its always good to test to make sure ... (I've been surprised before)

Reply:
I don't have IE on my mac yet, If you don't mind giving it a quick run through on IE/Edge that would be helpful